rules/internal/utils/extension.go

```diff
@@ -65,6 +65,20 @@ func GetResponseType(m *desc.MethodDescriptor) *desc.MessageDescriptor {
 	return typ
 }
 
+// GetResponseType returns the OutputType if the response is
+// not an LRO, or the ResponseType otherwise.
+func GetResponseType(m *desc.MethodDescriptor) *desc.MessageDescriptor {
+	if m == nil {
+		return nil
+	}
+
+	if m.GetOutputType().GetName() != "Operation" {
+		return m.GetOutputType()
+	}
```

Comment:
As mentioned in another comment, this probably isn't the best check. "Operation" doesn't necessarily correlate to LRO. In the interest of being robust, let's check if the output type is a `google.longrunning.Operation`, which obviously requires checking the type's package and the name of the message itself.

Reply:
ack, I added the package check.